diff --git a/node_modules/electron/electron.d.ts b/node_modules/electron/electron.d.ts
index 1998a92..6254df8 100644
--- a/node_modules/electron/electron.d.ts
+++ b/node_modules/electron/electron.d.ts
@@ -109,6 +109,7 @@ declare namespace Electron {
   const webFrame: WebFrame;
   const webviewTag: WebviewTag;
 
+  // @ts-ignore
   interface App extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/app
@@ -1513,6 +1514,7 @@ Here's a very simple example of creating a custom Jump List:
     userAgentFallback: string;
   }
 
+  // @ts-ignore
   interface AutoUpdater extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/auto-updater
@@ -1671,6 +1673,7 @@ Here's a very simple example of creating a custom Jump List:
     webContents: WebContents;
   }
 
+  // @ts-ignore
   class BrowserWindow extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/browser-window
@@ -3076,6 +3079,7 @@ This cannot be called when `titleBarStyle` is set to `customButtonsOnHover`.
     state: string;
   }
 
+  // @ts-ignore
   class ClientRequest extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/client-request
@@ -3463,6 +3467,7 @@ This cannot be called when `titleBarStyle` is set to `customButtonsOnHover`.
     stopRecording(resultFilePath?: string): Promise<string>;
   }
 
+  // @ts-ignore
   interface ContextBridge extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/context-bridge
@@ -3516,6 +3521,7 @@ This cannot be called when `titleBarStyle` is set to `customButtonsOnHover`.
     value: string;
   }
 
+  // @ts-ignore
   class Cookies extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/cookies
@@ -3686,6 +3692,7 @@ Sets a cookie with `details`.
     scheme: string;
   }
 
+  // @ts-ignore
   class Debugger extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/debugger
@@ -4239,6 +4246,7 @@ Send given command to the debugging target.
     show(): Promise<void>;
   }
 
+  // @ts-ignore
   class DownloadItem extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/download-item
@@ -4586,6 +4594,7 @@ If the size is unknown, it returns 0.
     webgl2: string;
   }
 
+  // @ts-ignore
   interface InAppPurchase extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/in-app-purchase
@@ -4635,6 +4644,7 @@ Retrieves the product descriptions.
     restoreCompletedTransactions(): void;
   }
 
+  // @ts-ignore
   class IncomingMessage extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/incoming-message
@@ -4741,6 +4751,7 @@ Retrieves the product descriptions.
     writeTransferCount: number;
   }
 
+  // @ts-ignore
   interface IpcMain extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/ipc-main
@@ -4831,6 +4842,7 @@ Retrieves the product descriptions.
     sender: WebContents;
   }
 
+  // @ts-ignore
   interface IpcRenderer extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/ipc-renderer
@@ -5223,6 +5235,7 @@ If you do not need a respons to the message, consider using `ipcRenderer.send`.
     visible: boolean;
   }
 
+  // @ts-ignore
   class MessageChannelMain extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/message-channel-main
@@ -5231,6 +5244,7 @@ If you do not need a respons to the message, consider using `ipcRenderer.send`.
     port2: MessagePortMain;
   }
 
+  // @ts-ignore
   class MessagePortMain extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/message-port-main
@@ -5444,6 +5458,7 @@ where `SYSTEM_IMAGE_NAME` should be replaced with any value from this list.
     isMacTemplateImage: boolean;
   }
 
+  // @ts-ignore
   interface NativeTheme extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/native-theme
@@ -5569,6 +5584,7 @@ Starts recording network events to `path`.
     newGuest?: BrowserWindow;
   }
 
+  // @ts-ignore
   class Notification extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/notification
@@ -5702,6 +5718,7 @@ Starts recording network events to `path`.
     y: number;
   }
 
+  // @ts-ignore
   interface PowerMonitor extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/power-monitor
@@ -6313,6 +6330,7 @@ e.g.
     username?: string;
   }
 
+  // @ts-ignore
   interface Screen extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/screen
@@ -6460,6 +6478,7 @@ e.g.
     scriptUrl: string;
   }
 
+  // @ts-ignore
   class ServiceWorkers extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/service-workers
@@ -6501,6 +6520,7 @@ e.g.
     getFromVersionID(versionId: number): ServiceWorkerInfo;
   }
 
+  // @ts-ignore
   class Session extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/session
@@ -7116,6 +7136,7 @@ Creates or updates a shortcut link at `shortcutPath`.
     mimeType?: string;
   }
 
+  // @ts-ignore
   interface SystemPreferences extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/system-preferences
@@ -7632,6 +7653,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     label: string;
   }
 
+  // @ts-ignore
   class TouchBarColorPicker extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-color-picker
@@ -7644,6 +7666,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     selectedColor: string;
   }
 
+  // @ts-ignore
   class TouchBarGroup extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-group
@@ -7654,6 +7677,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     constructor(options: TouchBarGroupConstructorOptions);
   }
 
+  // @ts-ignore
   class TouchBarLabel extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-label
@@ -7667,6 +7691,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     textColor: string;
   }
 
+  // @ts-ignore
   class TouchBarPopover extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-popover
@@ -7679,6 +7704,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     label: string;
   }
 
+  // @ts-ignore
   class TouchBarScrubber extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-scrubber
@@ -7695,6 +7721,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     showArrowButtons: boolean;
   }
 
+  // @ts-ignore
   class TouchBarSegmentedControl extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-segmented-control
@@ -7708,6 +7735,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     selectedIndex: number;
   }
 
+  // @ts-ignore
   class TouchBarSlider extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-slider
@@ -7722,6 +7750,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     value: number;
   }
 
+  // @ts-ignore
   class TouchBarSpacer extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-spacer
@@ -7843,6 +7872,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     transactionState: ('purchasing' | 'purchased' | 'failed' | 'restored' | 'deferred');
   }
 
+  // @ts-ignore
   class Tray extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/tray
@@ -8352,6 +8382,7 @@ This value is set to false by default.
     type: string;
   }
 
+  // @ts-ignore
   class WebContents extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/web-contents
@@ -10095,6 +10126,7 @@ Takes a V8 heap snapshot and saves it to `filePath`.
     zoomLevel: number;
   }
 
+  // @ts-ignore
   interface WebFrame extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/web-frame
@@ -14003,6 +14035,7 @@ interface Document {
 }
 
 declare namespace NodeJS {
+  // @ts-ignore
   interface Process extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/process
