diff --git a/node_modules/electron/electron.d.ts b/node_modules/electron/electron.d.ts
index e77fad7..b2213ef 100644
--- a/node_modules/electron/electron.d.ts
+++ b/node_modules/electron/electron.d.ts
@@ -1656,6 +1656,7 @@ Here's a very simple example of creating a custom Jump List:
     webContents: WebContents;
   }
 
+  // @ts-ignore
   class BrowserWindow extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/browser-window
@@ -3061,6 +3062,7 @@ This cannot be called when `titleBarStyle` is set to `customButtonsOnHover`.
     state: string;
   }
 
+  // @ts-ignore
   class ClientRequest extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/client-request
@@ -3501,6 +3503,7 @@ This cannot be called when `titleBarStyle` is set to `customButtonsOnHover`.
     value: string;
   }
 
+  // @ts-ignore
   class Cookies extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/cookies
@@ -3671,6 +3674,7 @@ Sets a cookie with `details`.
     scheme: string;
   }
 
+  // @ts-ignore
   class Debugger extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/debugger
@@ -4204,6 +4208,7 @@ Send given command to the debugging target.
     show(): Promise<void>;
   }
 
+  // @ts-ignore
   class DownloadItem extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/download-item
@@ -4600,6 +4605,7 @@ Retrieves the product descriptions.
     restoreCompletedTransactions(): void;
   }
 
+  // @ts-ignore
   class IncomingMessage extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/incoming-message
@@ -5464,6 +5470,7 @@ Starts recording network events to `path`.
     newGuest?: BrowserWindow;
   }
 
+  // @ts-ignore
   class Notification extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/notification
@@ -6351,6 +6358,7 @@ e.g.
     scriptUrl: string;
   }
 
+  // @ts-ignore
   class ServiceWorkers extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/service-workers
@@ -6392,6 +6400,7 @@ e.g.
     getFromVersionID(versionId: number): ServiceWorkerInfo;
   }
 
+  // @ts-ignore
   class Session extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/session
@@ -7519,6 +7528,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     label: string;
   }
 
+  // @ts-ignore
   class TouchBarColorPicker extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-color-picker
@@ -7531,6 +7541,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     selectedColor: string;
   }
 
+  // @ts-ignore
   class TouchBarGroup extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-group
@@ -7541,6 +7552,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     constructor(options: TouchBarGroupConstructorOptions);
   }
 
+  // @ts-ignore
   class TouchBarLabel extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-label
@@ -7554,6 +7566,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     textColor: string;
   }
 
+  // @ts-ignore
   class TouchBarPopover extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-popover
@@ -7566,6 +7579,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     label: string;
   }
 
+  // @ts-ignore
   class TouchBarScrubber extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-scrubber
@@ -7582,6 +7596,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     showArrowButtons: boolean;
   }
 
+  // @ts-ignore
   class TouchBarSegmentedControl extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-segmented-control
@@ -7595,6 +7610,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     selectedIndex: number;
   }
 
+  // @ts-ignore
   class TouchBarSlider extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-slider
@@ -7609,6 +7625,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     value: number;
   }
 
+  // @ts-ignore
   class TouchBarSpacer extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/touch-bar-spacer
@@ -7730,6 +7747,7 @@ This property is only available on macOS 10.14 Mojave or newer.
     transactionState: ('purchasing' | 'purchased' | 'failed' | 'restored' | 'deferred');
   }
 
+  // @ts-ignore
   class Tray extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/tray
@@ -8239,6 +8257,7 @@ This value is set to false by default.
     type: string;
   }
 
+  // @ts-ignore
   class WebContents extends NodeJS.EventEmitter {
 
     // Docs: http://electronjs.org/docs/api/web-contents
